### ISSUE TITLE
Current key layout widget

### ISSUE THREAD

**Issue by KryDos** [felixhageloh/uebersicht-widgets#267]
Hi, I'm new user of Ubersicht and I use different key layouts in my work. 
I used to see what is my current layout and unfortunately I didn't find any widget that can help me.

So, I wrote my own widget - https://github.com/KryDos/ubersicht-currentKeymap

Hope it can be useful. 

P.S
Have to mention that in my case OSX top bar (menu bar) is hidden and my widget shows in default menu bar of Ubersicht. Not sure, but probably it called NerdBar. 

P.P.S
I didn't notice some important rules about how to prepare widget for submit. So I added screenshot, widget.json and zip archive.

**Comment by felixhageloh** [comment]
Thanks for sharing and sorry for the late reply! I've added it! You might want to adjust you screenshot to the recommended size btw - right now it gets stretched quite a bit on the website

**Comment by KryDos** [comment]
@felixhageloh thanks for response and for accept of this widget. Yeah, it is stretched because I have pretty small screenshots that I added to my readme file. But I also have [screenshot.png](https://github.com/KryDos/ubersicht-currentKeymap/blob/master/screenshot.png) there which have exact size that described in the instruction. 

I also added it as a first one to my readme. But it looks like it is not reflected on uebersicht-widgets page. Should I do something else to fix this on uebersicht-widgets page? 

Thanks.
